src/test/java/com/exasol/projectkeeper/validators/ChangelogValidatorTest.java

```diff
@@ -20,17 +20,25 @@
 import org.junit.jupiter.api.io.TempDir;
 
 class ChangelogValidatorTest {
+    private static final String V_1_2_3 = "1.2.3";
 
     @Test
     void testValidation(@TempDir final File tempDir) {
-        final MavenProject project = createProject(tempDir);
+        final MavenProject project = createProject(tempDir, V_1_2_3);
+        assertThat(new ChangelogValidator(project),
+                hasValidationFindingWithMessage("E-PK-20 Could not find 'doc/changes/changes_1.2.3.md'."));
```

Comment:
I think we agreed to use : after the code?

Reply:
yes, we did but his one is older. (Even so the test is new...) So I will fix it with https://github.com/exasol/project-keeper-maven-plugin/issues/30